database/migrations/2019_07_26_160404_create_ipma_warnings_table.php

```diff
@@ -0,0 +1,43 @@
+<?php
```

Comment:
Let's get rid of this migration, in favour of fetching and just caching for 30min.